duckduckgo/iOS
# Improved WebKit termination pixels

Task/Issue URL: https://app.asana.com/0/856498667320406/1161842050800401
Tech Design URL:
CC:

**Description**:
Improved pixels, to better calculate impact of WebKit terminations on browsing experience.

**Steps to test this PR**:
1. Open the app and navigate between sites.
2. See that pixels are fired.


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
